server/app/app.test.js

```diff
@@ -0,0 +1,205 @@
+      },
+      {
+        id: 4,
+        name: "Tim's wishlist",
+        author: 'Tim',
+        items: [
+          { id: 7, wishlist_id: 4, name: 'People to stop calling him Tim-Tim' },
+          { id: 8, wishlist_id: 4, name: 'tissue box' },
+          { id: 9, wishlist_id: 4, name: 'plate' },
+        ],
+      },
+      {
+        id: 5,
+        name: "Dave's wishlist",
+        author: 'Dave',
+        items: [
+          { id: 10, wishlist_id: 5, name: 'remote' },
+          { id: 11, wishlist_id: 5, name: 'pair of socks' },
+          { id: 12, wishlist_id: 5, name: 'key' },
+        ],
+      },
+      {
+        id: 6,
+        name: "Sarah's wishlist",
+        author: 'Sarah',
+        items: [
+          { id: 13, wishlist_id: 6, name: 'coffee mug' },
+          { id: 14, wishlist_id: 6, name: 'tennis ball' },
+          { id: 15, wishlist_id: 6, name: 'tiger' },
+        ],
+      },
+    ];
+    it('should respond with a list of wishlists', async () => {
+      const response = await request(app).get('/wishlist');
+      expect(response.statusCode).toBe(200);
+      expect(response.body).toMatchObject(wishlists);
+    });
+  });
+
+  describe('GET /wishlist/:id', () => {
+    const wishlist = {
+      id: 4,
+      name: "Tim's wishlist",
+      author: 'Tim',
+      items: [
+        { id: 7, wishlist_id: 4, name: 'People to stop calling him Tim-Tim' },
+        { id: 8, wishlist_id: 4, name: 'tissue box' },
+        { id: 9, wishlist_id: 4, name: 'plate' },
+      ],
+    };
+    it('should respond with a wishlist with the same id', async () => {
+      const response = await request(app).get('/wishlist/4');
+      expect(response.statusCode).toBe(200);
+      expect(response.body).toMatchObject(wishlist);
+    });
+
+    it('should respond with a 500 if there was an error', async () => {
+      const response = await request(app).get('/wishlist/100');
+      expect(response.statusCode).toBe(500);
+      expect(response.body).not.toMatchObject(wishlist);
+    });
+  });
+
+  describe('POST /wishlist', () => {
+    it('should respond with a 201 and a message if successfully created', async () => {
+      const response = await request(app)
+        .post('/wishlist')
+        .send({
+          name: "Bob's wishlist",
+          author: 'Bob',
+          items: ['Socks', 'Christmas Socks', 'coal'],
+        });
+      expect(response.statusCode).toBe(201);
+      expect(response.body.message).toBe('Your wishlist has been created!');
+      expect(response.created).toBe(true);
+    });
+
+    it('should respond with a 500 if title is missing', async () => {
+      const response = await request(app)
+        .post('/wishlist')
+        .send({
+          author: 'Bob',
+          items: ['Socks', 'Christmas Socks', 'coal'],
+        });
+      expect(response.statusCode).toBe(500);
+      expect(response.created).toBe(false);
+    });
+
+    it('should respond with a 500 and a message if items are missing', async () => {
```

Comment:
Example in reference to my comment about sendStatus in server/app.js - here is one of the places we could make use of a message in the response either via sendStatus or by sending the error back.  Currently, our test is stated to be verifiying that the server `'should respond with a 500 and a message if items are missing'`, but we don't currently check for a message.